### DISCUSSION TITLE
How do I subscribe to Discussions?

### DISCUSSION THREAD

**Discussion by DmT021** [community/community#63521] [Discussions]
### Select Topic Area

Bug

### Body

It seems like there should be a button "Subscribe" or something on the right side of the screen, but there isn't.
![image](https://github.com/community/community/assets/1978725/1747a158-3673-489f-b590-1060ea9662a0)

**Comment by elderguardian** [comment]
Hi,

There's a website called [openrss](https://openrss.org/) that can make an RSS Feed from discussions. You can see how it works [here](https://github.com/orgs/community/discussions/31#discussioncomment-3807225). 
If you own the repository, you could try using the [discussion workflow event](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#discussion) on GitHub Actions.

**Comment by ramiroaraujo** [comment]
@DmT021 Just found this:
![image](https://github.com/community/community/assets/46276/e6a7fcea-8179-4bae-b9a3-efdf440216e9)
It seems you can `Watch` the repo with a custom selection of events, and in my case I also only want to be notified of Discussions. I've just set it, we'll see how it works :)

**Reply by DmT021** [reply]
Wow. I think this is it. Thanks.

**Reply by DmT021** [reply]
Although it seems strange. "Discussions" is an entity of Organisation, but "watch" is an action on a repo

**Comment by dakshithadissanayaka** [comment]
Subscribing to Discussions on GitHub allows you to receive notifications for new comments, replies, and updates in specific discussions or organizations. Here's a step-by-step guide on how to subscribe to Discussions on GitHub:

1. **Navigate to the Discussion:** Open the specific discussion or organization page where you want to enable notifications.

2. **Locate the Subscription Button:** Look for the "Subscribe" button located near the top of the discussion thread or organization page. It may be next to the "Watch" button or represented by a bell icon.

3. **Click the "Subscribe" Button:** Clicking the "Subscribe" button will enable notifications for that particular discussion or organization. You'll receive notifications for new comments, replies, and updates in your email and GitHub notifications.

4. **Customize Notification Preferences (Optional):** If you want to customize your notification preferences, click the small dropdown arrow next to the "Subscribe" button. This will allow you to choose whether to receive notifications for all activity or only for replies to your comments.

5. **Manage Subscriptions (Optional):** To manage your existing subscriptions, click the "Manage notifications" dropdown menu located in the left sidebar of your GitHub profile page. This will show a list of all the discussions and organizations you're subscribed to, and you can unsubscribe from any of them by clicking the "Unsubscribe" button next to each subscription.

By subscribing to Discussions on GitHub, you'll stay up-to-date with the latest conversations and happenings in the communities you care about.

**Comment by Realiserad** [comment]
It would be great if you can subscribe to discussions with a specific label or category to reduce noise.

**Reply by BigLep** [reply]
Agreed.  Other times this has been brought up:
https://github.com/orgs/community/discussions/20316
https://github.com/orgs/community/discussions/3951

**Comment by Artur-phd** [comment]
![image](https://github.com/user-attachments/assets/a06e5317-f1ac-42b2-962a-fb814cc15f46)
this is weird...

**Comment by deenudubai5** [comment]
You can usually subscribe to discussions by clicking the "Follow" or "Subscribe" button on the thread or in your account settings [Dubai Khaleej](https://dubaikhaleej.com/). Check if there’s an option for email notifications to stay updated on new replies.

**Reply by robross0606** [reply]
That subscribes to one specific discussion thread.  The conversation is around subscribing to an entire category of discussion threads.

**Comment by SilentGlasses** [comment]
Did you get this answered yet @DmT021 ?  If not.. 
discussion post, look for the **Notifications** section, there should be a button named `Subscribe`.

See the bottom right of the screenshot from this very Discussion to see what the section and button looks like:
![image](https://github.com/user-attachments/assets/48b19b1b-4c30-41cb-a293-cc001cedfd12)
